### PR TITLE
fix(locksmith): optimise tests

### PR DESCRIPTION
**Pull request by 0xTxbi**
# Description
This PR introduces updates to our test suites, enabling them to run in parallel without cross-test pollution

# Issues
Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/__tests__/controllers/v2/priceController.test.ts [hunk 1]
```diff
@@ -1,58 +1,68 @@
...
 import { networks } from '@unlock-protocol/networks'
+import { expect, describe, it, beforeEach, afterEach } from 'vitest'
 
 import app from '../../app'
 
 const networkIds = Object.values(networks)
   .filter((network) => !!network.tokens)
   .map((item) => item.id)
 
-describe.each(networkIds)('Test the price on each network', async (id) => {
```
Comment (julien51) [inline locksmith/__tests__/controllers/v2/priceController.test.ts:10]: I think we need to use that

File: locksmith/__tests__/controllers/v2/ticketsController.test.ts [hunk 2]
```diff
@@ -246,20 +255,20 @@ describe('tickets endpoint', () => {
         }),
         headers: { 'Content-Type': 'application/json' },
         method: 'POST',
-        signal: expect.any(AbortSignal),
+        signal: expect.any(Object),
```
Comment (julien51) [inline locksmith/__tests__/controllers/v2/ticketsController.test.ts:258]: hum, that does not seem right?

**Review by julien51 — COMMENTED** [review]
Please do the extra changes but LGTM!

File: locksmith/__tests__/controllers/v2/priceController.test.ts (hunk 1 above)
Comment (0xTxbi) [reply locksmith/__tests__/controllers/v2/priceController.test.ts:10]: reverted!

File: locksmith/__tests__/controllers/v2/ticketsController.test.ts (hunk 2 above)
Comment (0xTxbi) [reply locksmith/__tests__/controllers/v2/ticketsController.test.ts]: fixed!

**Review by julien51 — APPROVED** [review]
LGTM!

**Pull request merged**